_data/sites.json

```
@@ -1199,6 +1199,19 @@
         ]
     },
 
+    {
+        "name": "Amso",
+        "url": "https://amso.eu/en/contact",
+        "difficulty": "hard",
+        "notes": "You need to write at contact@amso.eu and ask them to delete your account. After 1 day they confirmed the deletion.",
```

```suggestion
        "notes": "You need to write an email to ask them to delete your account. After 1 day the deletion is confirmed.",
```
We tend not to add the e-mail address on the notes, since it already has a dedicated field to it.